docs/app/references/launching-browsers.mdx

```diff
@@ -190,6 +190,20 @@ of our [docker images](/app/continuous-integration/overview#Cypress-Docker-Image
 By default, we will launch Firefox headlessly during `cypress run`. To run
 Firefox headed, you can pass the `--headed` argument to `cypress run`.
 
+##### Webdriver BiDi and CDP Deprecation
+
+:::info
+
+Since Firefox 129, Chrome DevTools Protocol (CDP) has been [deprecated in Firefox](https://fxdx.dev/deprecating-cdp-support-in-firefox-embracing-the-future-with-webdriver-bidi/). Cypress will default to automating the Firefox browser with WebDriver BiDi in Firefox 135. Cypress will no longer support CDP within Firefox in the future. Cypress CDP support within Firefox is planned to be removed in Cypress 15.
```

Comment:
Suggestion for a rewording which also works for later:

In Firefox 135 and above, Cypress defaults to automating the Firefox browser with WebDriver BiDi.

Reply:
Also, if you put each sentence on a new line it makes editing / git differencing a bit simpler.
Docusaurus ignores the new lines and put everything into one paragraph.

Reply:
@MikeMcC399 good to know! I applied your suggestions in 429f36f56